src/Controls/src/Core/Layout/Layout.cs

```diff
@@ -324,5 +325,24 @@ void UpdateDescendantInputTransparent()
 				}
 			}
 		}
+
+		internal static void MapIsEnabled(ILayoutHandler handler, Layout layout)
+		{
+			layout.UpdateDescendantIsEnabled();
+		}
+
+		void UpdateDescendantIsEnabled() 
+		{
+			var isEnabled = IsEnabled;
+
+			// Set all the child IsEnabled values to match this one
+			for (int n = 0; n < Count; n++)
+			{
+				if (this[n] is VisualElement visualElement)
+				{
+					visualElement.IsEnabled = isEnabled;
```

Comment:
Can we modify the properties like this? Won't this remove any bindings the user has defined for `IsEnabled` on any child elements?

Reply:
It would bypass those bindings, yes. The next question would be whether that's a concern. 

If it is, then we have a couple of options:

- We could determine if there's a way to check whether the property was bound before setting it, and skip it in that case. 
- We could go with the Forms behavior of simply disabling input on the Layout/ContentView, with the attendant problems (no visual indication that child controls are "disabled", etc.)